src/main/java/org/jitsi/protocol/xmpp/util/MediaSourceGroupMap.java

```diff
@@ -70,6 +124,32 @@ private MediaSourceGroupMap(Map<String, List<SourceGroup>> map)
     }
 
     /**
+     * Finds groups that match given media type and semantics.
+     *
+     * @param media eg. 'audio', 'video', etc.
+     * @param semantics a group semantics eg. 'SIM' or 'FID'
+     *
+     * @return a {@link List} of {@link SourceGroup}
+     */
+    private List<SourceGroup> findSourceGroups(String media, String semantics)
+    {
+        Objects.requireNonNull(semantics, "semantics");
+
+        List<SourceGroup> mediaGroups = groupMap.get(media);
+        List<SourceGroup> result = new LinkedList<>();
+
+        for (SourceGroup group : mediaGroups)
```

Comment:
same comment as below

Reply:
I will keep this one as is, because it's generic and doesn't mention it's about video kind of groups (just an example in arg description). We don't know if there will be no groups for audio.